back/engines/free/email_campaigns/app/mailers/email_campaigns/application_mailer.rb

```diff
@@ -38,12 +40,40 @@ def show_privacy_policy_link?
       true
     end
 
+    # To format a non-editable message, use `format_message`.
     def format_message(key, component: nil, escape_html: true, values: {})
       group = component || @campaign.class.name.demodulize.underscore
       msg = t("email_campaigns.#{group}.#{key}", **values)
       escape_html ? msg : msg.html_safe
     end
 
+    # To format an editable message, use `format_editable_region`.
+    # The `region_key` must exist in editable regions.
+    def format_editable_region(region_key: nil, values: {})
+      return unless region_key
+      return Error unless self.class.editable_regions.any? { |r| r[:key] == region_key }
+
+      # TODO: What about HTML escaping? Should we always escape?
```

Comment:
Perhaps we should use a sanitizer for custom copy? Or we could consider just trusting the moderators 😅